tests/test_experiment_rsmpredict.py

```diff
@@ -408,6 +407,35 @@ def test_fast_predict_no_feature_info(self):
         with self.assertRaises(ValueError):
             _ = fast_predict(input_features, modeler)
 
+    def test_fast_predict_feature_info_none(self):
+        """Check case where there is feature information is ``None``."""
+        existing_experiment_dir = join(
+            rsmtool_test_dir,
+            "data",
+            "experiments",
+            "lr-predict",
+            "existing_experiment",
+            "output",
+        )
+        model_file = join(existing_experiment_dir, "lr.model")
+
+        # initialize the modeler instance with a bare learner, in which
+        # case no feature_info attribute will exist
+        modeler = Modeler.load_from_learner(Modeler.load_from_file(model_file).learner)
```

Comment:
What I wanted to achieve with this line was to load from the learner, but in order to load from the bare learner I first have to load from the modeler and then get its learner attribute. Then, I can reload from that learner attribute as if I had loaded from a learner from the start. Technically, this is done only once. There are similar lines in 2 other locations, but in those cases the calls are not chained together like this (since I need to get some other info from the modeler before loading from the learner). I don't think it's worth it to abstract this out into a utility function. In fact, I think it would just add complexity.